examples/Viewer.tsx

```diff
@@ -603,7 +622,6 @@ class Viewer extends React.Component<{}, ViewerState> {
                         )}
                         loadInitialData={true}
                         agentColors={this.state.agentColors}
-                        hideAllAgents={this.state.hideAllAgents}
```

Comment:
future work: remove this prop from the component?  I don't think we would use it when we can do it using the Selection interface instead.